controller/cmd/destination/main.go

```diff
@@ -24,6 +24,7 @@ func Main(args []string) {
 
 	addr := cmd.String("addr", ":8086", "address to serve on")
 	metricsAddr := cmd.String("metrics-addr", ":9996", "address to serve scrapable metrics on")
+	policyAddr := cmd.String("policy-addr", ":8090", "address to connect to the policy server")
```

Comment:
```suggestion
	policyAddr := cmd.String("policy-addr", "127.0.0.1:8090", "address to connect to the policy server")
```

I don't think we can use `:8090` as a client's target address.